conf/carbon.conf.example

```diff
@@ -262,6 +262,14 @@ RELAY_METHOD = rules
 # datapoint to more than one machine.
 REPLICATION_FACTOR = 1
 
+# If your REPLICATION_FACTOR >=2, you are using more than one replica per server
+# and you want to be sure that every replica of data is located on different servers
+# please setup DIVERSE_REPLICAS to True.
+# Before carbon v0.9.14 it was not guaranteed!
```

Comment:
I would prefer that we give an "active" explanation of what this feature actually does, rather than implying how it might or might not be different from past behavior.

Also, rather than saying `Before carbon v0.9.14`, let's describe what the past behavior would be in the context of `DIVERSE_REPLICAS` being `True` or `False`.